src/Service.Tests/dab-config.CosmosDb_NoSql.json

```diff
@@ -1,258 +1,655 @@
 {
-  "$schema": "../../schemas/dab.draft.schema.json",
+  "$schema": "https://github.com/Azure/data-api-builder/releases/download/vmajor.minor.patch/dab.draft.schema.json",
   "data-source": {
     "database-type": "cosmosdb_nosql",
+    "connection-string": "AccountEndpoint=https://localhost:8081/;AccountKey=C2y6yDjf5/R\u002Bob0N8A7Cgv30VRDJIWEHLM\u002B4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==",
     "options": {
       "database": "graphqldb",
       "container": "planet",
       "schema": "schema.gql"
-    },
-    "connection-string": "AccountEndpoint=https://localhost:8081/;AccountKey=C2y6yDjf5/R\u002Bob0N8A7Cgv30VRDJIWEHLM\u002B4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw=="
+    }
   },
   "runtime": {
     "rest": {
-      "enabled": true,
+      "enabled": false,
       "path": "/api"
     },
     "graphql": {
-      "allow-introspection": true,
       "enabled": true,
-      "path": "/graphql"
+      "path": "/graphql",
+      "allow-introspection": true
     },
     "host": {
-      "mode": "development",
       "cors": {
         "origins": [
           "http://localhost:5000"
         ],
         "allow-credentials": false
       },
       "authentication": {
-        "provider": "StaticWebApps"
-      }
+        "provider": "StaticWebApps",
+        "jwt": {
+          "audience": null,
+          "issuer": null
+        }
+      },
+      "mode": "development"
     }
   },
   "entities": {
     "Planet": {
-      "source": "graphqldb.planet",
-      "permissions": [
-        {
-          "role": "anonymous",
-          "actions": [
-            "create",
-            {
-              "action": "read",
-              "fields": {
-                "include": [
-                  "*"
-                ],
-                "exclude": []
-              }
-            },
-            "update",
-            "delete"
-          ]
-        },
-        {
-          "role": "authenticated",
-          "actions": [
-            "create",
-            "read",
-            "update",
-            "delete"
-          ]
-        }
-      ],
-      "graphql": {
-        "type": {
-          "singular": "Planet",
-          "plural": "Planets"
+  "source": {
+    "object": "graphqldb.planet",
+    "type": null,
+    "parameters": null,
```

Comment:
If the keyvalue is `null`, shouldnt we skip writing it to the file? If you notice, the CosmosDb_NoSql config file now is filled with bunch of nulls which makes it hard to read. Keeping the config simple/readable is one of the goals we should try to achieve. One option to fix this could be in a followup PR - since we would like this to be checked in to as early as possible unblock others

Reply:
That's because https://github.com/Azure/data-api-builder/pull/1529 hasn't been merged in yet (it can either be merged with the branch of this PR or merged afterwards).

Looks like the build is failing for #1529 though so I'll fix that up so it's green and ready for review.

Reply:
I had forgotten about #1529.